src/navigation/Routes.ios.tsx

```diff
@@ -326,10 +328,11 @@ function NativeStackNavigator() {
         {...restoreSheetConfig}
       />
       <NativeStack.Screen
-        component={TransactionConfirmationScreen}
+        component={SignTransactionSheet}
         name={Routes.CONFIRM_REQUEST}
         options={{
-          allowsDragToDismiss: true,
+          allowsDragToDismiss: false,
```

Comment:
we are no longer allowing dismissal of WC requests as a pending request will block any new incoming requests